neuralmonkey/config/builder.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, string):
 
     def create(self):
         class_parts = self.clazz.split(".")
+
```

Comment:
Nešlo by uložit `class_name` a `class_module` do `self`? A nešlo by použít něco nativnějšího, jako třeb `clazz.__module__` a `clazz.__name__`?